server.js

```diff
@@ -5634,30 +5634,25 @@ cache(function(data, match, sendBadge, request) {
   var badgeData = getBadgeData('website', data);
   badgeData.colorscheme = undefined;
   request(options, function(err, res) {
-    try {
-      // We consider all HTTP status codes below 310 as success.
-      if (err != null || res.statusCode >= 310) {
-        badgeData.text[1] = offlineMessage;
-        if (sixHex(offlineColor)) {
-          badgeData.colorB = '#' + offlineColor;
-        } else {
-          badgeData.colorscheme = offlineColor;
-        }
-        sendBadge(format, badgeData);
-        return;
+    // We consider all HTTP status codes below 310 as success.
+    if (err != null || res.statusCode >= 310) {
+      badgeData.text[1] = offlineMessage;
+      if (sixHex(offlineColor)) {
+        badgeData.colorB = '#' + offlineColor;
       } else {
-        badgeData.text[1] = onlineMessage;
-        if (sixHex(onlineColor)) {
-          badgeData.colorB = '#' + onlineColor;
-        } else {
-          badgeData.colorscheme = onlineColor;
-        }
-        sendBadge(format, badgeData);
-        return;
+        badgeData.colorscheme = offlineColor;
       }
-    } catch(e) {
-      badge({text: ['error', 'bad badge'], colorscheme: 'red'},
-        makeSend(format, ask.res, end));
+      sendBadge(format, badgeData);
+      return;
+    } else {
+      badgeData.text[1] = onlineMessage;
+      if (sixHex(onlineColor)) {
+        badgeData.colorB = '#' + onlineColor;
+      } else {
+        badgeData.colorscheme = onlineColor;
+      }
+      sendBadge(format, badgeData);
+      return;
```

Comment:
I don't see code that is meant to trigger an exception. There is no parsing or pulling out keywords. The `catch` block never would have functioned correctly due to undefined variables inside it, so it seems simplest to remove it.